R/drop_pmid_version.R

```diff
@@ -38,8 +38,14 @@ deleteOldPmidVersions = function(tableSuffix, dryRun, dbtype, dbname, ...) {
     DBI::dbRemoveTable(con, tableKeep)
   } else {
     DBI::dbRemoveTable(con, tableNow)
-    q = glue_sql('alter table {`tableKeep`} rename to {`tableNow`}', .con = con)
-    n = DBI::dbExecute(con, q)}
+    if(inherits(con, 'BigQueryConnection')){
```

Comment:
Clean up whitespace in places like this.

Reply:
Could you explain how you want the whitespace to be?

Reply:
use `if (...) {`, not `if(...){`